src/features/auth/token.ts

```diff
@@ -0,0 +1,10 @@
+let token: string | undefined;
```

Comment:
is this a global, or whats the context of where this variable is now stored?

Reply:
this is not exactly a global, in the sense that you can only access it from outside this file with the exported setter & getter, but it is a singleton. this isn't a change from previous behavior, because the old ApiService was a singleton.